### DISCUSSION TITLE
2024-03-06

### DISCUSSION THREAD

**Discussion by yutingzhao1991** [ant-design/ant-design-web3#593] [中文社区双周会]
上一次双周会会议纪要：

https://github.com/ant-design/ant-design-web3/discussions/576#discussioncomment-8540532

**Comment by yutingzhao1991** [comment]
@jeasonstudio 明天的会有要讨论和对齐的议题可以先整理回复到这里。

**Comment by jeasonstudio** [comment]
本次议题：
* 体验优化和走查（参考新交互稿）：
  * https://github.com/ant-design/ant-design-web3/issues/386
  * https://github.com/ant-design/ant-design-web3/issues/550
  * https://github.com/ant-design/ant-design-web3/issues/552
  * 钱包交互方式优化，插件和二维码这块交互看看能不能优化（文卿、 @LCJove  ）
* `solana` 适配器发 1.0 前的其他工作：
  * 完善 sub pkg readme 和文档 @gin-lsl
  * 默认提供更多的钱包配置（OKX、Trust Wallet 等）https://solana.com/ecosystem/explore?categories=wallet
  * 尝试和 solana 开发者社区联系，获得更大的曝光 @jeasonstudio https://github.com/solana-foundation/developer-content/blob/main/content/guides/getstarted/hello-world-in-your-browser.md
* 文档切换到 antd 主题：https://kuangpf.com/dumi-theme-antd/ @jeasonstudio 
* antd 默认主题太过常规，提供一套更大胆、更前卫、更 web3 的 design token，以后做设计和实现时默认要关注两套 UI 的效果（ 复用 zan 等现成的 design token 快速搞几套，官网中支持切换预览，够炫酷；尝试把 theme-editor 拿过来作为主题编辑器） @gin-lsl 
* USDT 充值组件进展讨论
* Swap 组件讨论
* ABI 展示组件讨论

暂时梳理了这些，@yutingzhao1991 看看有没有补充，我稍后把这些条目转换为 issue 跟进

**Reply by yutingzhao1991** [reply]
还有 https://github.com/ant-design/ant-design-web3/pull/591 这个 PR 也过一下，看看还需要改什么，这个也是一个重要功能更新

**Comment by jeasonstudio** [comment]
补充：
* 讨论如何集成 SIWE（Sign-In with Ethereum，https://docs.login.xyz/ ） @jeasonstudio 
* 希望能保持 icon 周期性固定新增，讨论如何建立机制或 SOP 保障

**Reply by jeasonstudio** [reply]
* Solana 钱包 top5
* https://coinmarketcap.com/ 各个品类 top 20
*

**Comment by jeasonstudio** [comment]
目前想到的宣传重点：
1. 课程（建设中）
2. 支持 eip6963（多个钱包插件避免冲突）和交互 @yutingzhao1991 
3. Solana 兼容 @jeasonstudio  @gin-lsl 
4. 新 icon

**Reply by yutingzhao1991** [reply]
Ant Design Web3 的课程其实已经算弄完了。

**Reply by yutingzhao1991** [reply]
丰富的 icons，这个也要讨论一下，要持续的把 icon 完善起来

**Comment by yutingzhao1991** [comment]
- 需求->设计->开发的节奏，看看怎么快速迭代提升组件整体质量
- 支付组件功能讨论 @Likang0122

**Comment by yutingzhao1991** [comment]
Solana 适配器的 API 再对一下，看看能不能再简化，后面要发个 1.0 版本。

**Reply by yutingzhao1991** [reply]
![image](https://github.com/ant-design/ant-design-web3/assets/1061968/f45be347-38f8-4cc4-a81e-c64e0cadadfb)

另外 autoConnect 是什么意思？

**Comment by yutingzhao1991** [comment]
讨论下一句话描述：https://github.com/ant-design/ant-design-web3/issues/611

**Reply by jeasonstudio** [reply]
discussion 异步跟进
